src/components/MeetSearchInput.tsx

```diff
@@ -1,41 +1,74 @@
+import { useState } from 'react';
 import { View, TextInput, StyleSheet } from 'react-native';
 import SearchIcon from '../../assets/search_icon.svg';
+import { TempMeetList } from '../data/TempMeetList';
+import { Meet } from '../screens/MeetSearchScreen';
+
+interface inputStyleProps {
+  isMarginTop: boolean;
+  setResultList: React.Dispatch<React.SetStateAction<Meet[]>>;
+  setIsSearch: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const MeetSearchInput = ({
+  isMarginTop,
+  setResultList,
+  setIsSearch,
+}: inputStyleProps) => {
+  const [inputText, setInpuText] = useState('');
+
+  const handleSubmit = () => {
+    if (!inputText) {
+      setResultList([]);
+      setIsSearch(false);
+      return;
+    }
+    const temp = TempMeetList.filter(item => item.meetName.includes(inputText));
+    setResultList(temp);
+    setIsSearch(true);
+  };
 
-const MeetSearchInput = () => {
   return (
-    <View style={styles.topContainer}>
-      <View style={styles.wrapper}>
-        <SearchIcon style={styles.searchIcon} />
-        <TextInput style={styles.searchInput} placeholder='Meet 검색하기...' />
+    <View style={styles(isMarginTop).topContainer}>
```

Comment:
이거보다는 조건부가 나은 렌더링 같아요.

```suggestion
    <View style={isMarginTop && styles.topContainer}>
```